scss/themes/_branded.scss

```diff
@@ -68,5 +62,14 @@
 @mixin oTopperHeadshotImage {
 	max-width: 100%;
 	display: block;
-	margin-left: 8px;
+}
+
+@mixin oTopperHeadshotSpacing {
+	$space-between-text-and-headshot: oTypographySpacingSize(2);
```

Comment:
I'd argue that this is the only part that is overkill—you can use oTypographySpacingSize(2) directly on the properties.

Reply:
nice, was on the fence with that 👌